alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestToggleMonitorAction.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.resthandler
+
+import org.apache.logging.log4j.LogManager
+import org.opensearch.alerting.AlertingPlugin
+import org.opensearch.client.node.NodeClient
+import org.opensearch.common.xcontent.XContentType
+import org.opensearch.commons.alerting.action.AlertingActions.TOGGLE_MONITOR_ACTION_TYPE
+import org.opensearch.commons.alerting.action.ToggleMonitorRequest
+import org.opensearch.commons.alerting.action.ToggleMonitorResponse
+import org.opensearch.commons.alerting.model.Monitor
+import org.opensearch.commons.alerting.util.AlertingException
+import org.opensearch.core.action.ActionListener
+import org.opensearch.core.rest.RestStatus
+import org.opensearch.core.xcontent.ToXContent
+import org.opensearch.core.xcontent.XContentBuilder
+import org.opensearch.rest.BaseRestHandler
+import org.opensearch.rest.BytesRestResponse
+import org.opensearch.rest.RestHandler
+import org.opensearch.rest.RestHandler.ReplacedRoute
+import org.opensearch.rest.RestHandler.Route
+import org.opensearch.rest.RestRequest
+import org.opensearch.rest.RestRequest.Method.PUT
+import java.io.IOException
+
+private val log = LogManager.getLogger(RestToggleMonitorAction::class.java)
+
+class RestToggleMonitorAction : BaseRestHandler() {
+    override fun getName(): String {
+        return "toggle_monitor_action"
+    }
+
+    override fun routes(): List<Route> {
+        return listOf()
+    }
+
+    override fun replacedRoutes(): MutableList<RestHandler.ReplacedRoute> {
+        return mutableListOf(
+            ReplacedRoute(
+                PUT,
+                "${AlertingPlugin.MONITOR_BASE_URI}/{monitorID}/_enable",
```

Comment:
What security level perms are we giving to these APIs, we need to make changes in Security plugin for that, check this -> https://github.com/opensearch-project/security/blob/a1ccbd56621609d1ae0b0f7b4780b968ed06024b/config/roles.yml#L32